<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loing</class>
 <widget class="QMainWindow" name="Loing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>343</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Loing</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(46, 47, 48);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_pic">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="userLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userEdit">
        <property name="styleSheet">
         <string notr="true">background-color: #3e3e3e</string>
        </property>
        <property name="maxLength">
         <number>64</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="passLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Password: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passEdit">
        <property name="styleSheet">
         <string notr="true">background-color: #3e3e3e</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Show password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">background-color: #333333;</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="loginButton">
      <property name="styleSheet">
       <string notr="true">background-color: #333333;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>checkBox</zorder>
   <zorder>loginButton</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>passEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
